valhalla/baldr/pathlocation.h

```diff
@@ -228,6 +229,14 @@ struct PathLocation : public Location {
       l.search_filter_.exclude_bridge_ = loc.search_filter().exclude_bridge();
       l.search_filter_.exclude_ramp_ = loc.search_filter().exclude_ramp();
       l.search_filter_.exclude_closures_ = loc.search_filter().exclude_closures();
+      if (loc.search_filter().has_level()) {
+        l.search_filter_.level_ = loc.search_filter().level();
+        if (loc.has_search_cutoff_case()) {
+          // set smaller default and add warning
+          l.search_cutoff_ = 300;
+          add_warning(request, 401, std::to_string(300) + " meters");
```

Comment:
Not sure yet about this, this would add one warning per location. When serializing out to json, we could dedupe these.